### ISSUE TITLE
autopopulate not passed to pool attachment

### ISSUE THREAD

**Issue by DavidGamba** [terraform-providers/terraform-provider-bigip#242]
I have a node defined with the following information:

```
  ~ resource "bigip_ltm_node" "this" {
        address          = "alb-tst.awstst.com"
        connection_limit = 0
        description      = "WEB ALB"
        dynamic_ratio    = 1
        id               = "/Common/alb-tst.awstst.com"
        monitor          = "default"
        name             = "/Common/alb-tst.awstst.com"
        rate_limit       = "disabled"
        ratio            = 1
        state            = "user-up"

      ~ fqdn {
            autopopulate = "enabled"
            downinterval = 5
            interval     = "60"
          + name         = "alb-tst.awstst.com"
        }
    }
```

It points to an AWS ALB with 2 IPs.

When I attach it to the pool:

```
  + resource "bigip_ltm_pool_attachment" "this" {
      + id   = (known after apply)
      + node = "/Common/alb-tst.awstst.com:80"
      + pool = "/Common/dgamba_test_80"
    }
```

The autopopulate enabled is not set on the member, so it only has 1 _auto_IP.

![image](https://user-images.githubusercontent.com/1115287/73205689-c63d5300-40fe-11ea-81bb-8f12b8290550.png)

If I attach it manually by selecting `FQDN Node List` the FQDN Auto Populate option already defaults to Enabled and it populates two _auto_IPs.

![new-pool-member-fqdn-node-list](https://user-images.githubusercontent.com/1115287/73206711-a9a21a80-4100-11ea-8531-238d03892b29.png)

Is autopopulate supported at the moment?

```
Terraform v0.12.20
+ provider.aws v2.46.0
+ provider.bigip v1.1.1

BIG-IP 13.1.0.8 Build 0.0.3 Point Release 8
```

**Comment by papineni87** [comment]
By default when we attach node to a pool through icontrol rest call , autopopulate will be disabled unless we specify it as enabled. we have to change the code to explicitly add fqdn  structure to rest call when adding node to pool. 


**curl request without autopopulate:  ( current behaviour )**

root@terraforn-ubuntu3:~/go/src/github.com/terraform-providers/terraform-provider-bigip# curl -sk -u xxxx:xxxx -H "Content-Type: application/json" -X POST https://10.145.69.41/mgmt/tm/ltm/pool/test_pool/members -d '{"name":"/Common/alb-tst.awstst.com:80"}'

**response:**

{"kind":"tm:ltm:pool:members:membersstate","name":"alb-tst.awstst.com:80","partition":"Common","fullPath":"/Common/alb-tst.awstst.com:80","generation":3271,"selfLink":"https://localhost/mgmt/tm/ltm/pool/test_pool/members/~Common~alb-tst.awstst.com:80?ver=13.1.0","address":"any6","connectionLimit":0,"dynamicRatio":1,"ephemeral":"false","fqdn":{"autopopulate":"disabled","tmName":"alb-tst.awstst.com"},"inheritProfile":"enabled","logging":"disabled","monitor":"default","priorityGroup":0,"rateLimit":"disabled","ratio":1,"session":"user-enabled","state":"fqdn-up-no-addr"}


**curl request with autopopulate** 

root@terraforn-ubuntu3:~/go/src/github.com/terraform-providers/terraform-provider-bigip# curl -sk -u xxxxx:xxxx -H "Content-Type: application/json" -X POST https://10.145.69.41/mgmt/tm/ltm/pool/test_pool/members -d '{"name":"/Common/alb-tst.awstst.com:80","fqdn":{"autopopulate":"enabled","tmName":"alb-tst.awstst.com"}}'


result:

{"kind":"tm:ltm:pool:members:membersstate","name":"alb-tst.awstst.com:80","partition":"Common","fullPath":"/Common/alb-tst.awstst.com:80","generation":3431,"selfLink":"https://localhost/mgmt/tm/ltm/pool/test_pool/members/~Common~alb-tst.awstst.com:80?ver=13.1.0","address":"any6","connectionLimit":0,"dynamicRatio":1,"ephemeral":"false","fqdn":{"autopopulate":"enabled","tmName":"alb-tst.awstst.com"},"inheritProfile":"enabled","logging":"disabled","monitor":"default","priorityGroup":0,"rateLimit":"disabled","ratio":1,"session":"user-enabled","state":"fqdn-up-no-addr"}

**Comment by DavidGamba** [comment]
@papineni87 thanks for getting back to me. Is there a similar way to also set a `Member Specific` monitor during attachment. That would really help too.

**Comment by georambo** [comment]
Hi, any update on this issue, was this ever merged?  I've tried the following fqdn block, and it is not setting the autopopulate flag as enabled.

 * provider.bigip: version = "~> 1.1"


```
fqdn {
    autopopulate = "enabled"
    downinterval = 5
    interval     = "60"
    name         = "xyz.com"
  }
```

**Comment by focrensh** [comment]
Tracking with TER-308 Internally.

**Comment by RavinderReddyF5** [comment]
Fixed in v1.2.0
